### PR TITLE
Fixes#414:AskSusi EditText and Mic Hides on search

### PR DESCRIPTION
**Pull request by HimanshuS01**
Fixes issue #414 

Changes: AskSusiSomething EditText and Mic hides on tapping the search icon.

Screenshots for the change: 

device-2016-12-26-232608

### CODE REVIEW

**Comment by HimanshuS01** [comment]
Hey @daminisatya @jig08 while resolving the merge conflict through github text editor,It results into the one more commit. Is it fine or I need to merge them into a single commit?Thanks

**Comment by samsruti** [comment]
@HimanshuS01 Yes we need to squash all commits into one. Only after resolving them, this PR can be merged.
Thanks.

**Comment by daminisatya** [comment]
Please resolve the conflicts and it's good to go

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -265,7 +265,6 @@ private void promptSpeechInput() {
     }
 
 
-
```
Comment (daminisatya) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Make sure only your changes get in. It's hard for reviewing these spacing and indentation changes.

**Comment by HimanshuS01** [comment]
@daminisatya  I have resolved the colflicts.Kindly review it.Thanks.

**Comment by mariobehling** [comment]
Circle CI tests are failing. Please check.

**Comment by samsruti** [comment]
@HimanshuS01 Please Check. 
/app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java:115: error: @BindView fields must not be private or static. (org.fossasia.susi.ai.activities.MainActivity.dates)
    private TextView dates;
You need to fix this to make the Circle Ci test to be passed.
Thanks :)

**Comment by HimanshuS01** [comment]
@mariobehling @daminisatya Thanks for reviewing it .I have made the requested changes in my PR.

There is an **authentication** error occuring in ci/circleci test,Rest seems to be fine.

**Error in ci/circleci test**
 Authentication failed for 'https://the-dagger:@github.com/fossasia/susi_android/'

**Comment by HimanshuS01** [comment]
@the-dagger @jig08 @daminisatya please review the PR. Thanks.

**Pull request merged**